kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -129,6 +127,7 @@
       creatingUser: false,
       editingUser: false,
       currentUserEdit: null,
+      userKinds : require('kolibri.coreVue.vuex.constants').UserKinds,
```

Comment:
I'd recommend attaching it to a computed property because these are read-only. See e.g.: https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/learn/assets/src/vue/content-page/index.vue#L53